README.md

```diff
@@ -38,6 +38,9 @@ There are two options:
     -  #### [Anonymization using ASR-BN with vector quantization (VQ)](https://arxiv.org/abs/2308.04455): **B5** and **B6** 
 
         [`configs/anon_asrbn.yaml`](configs/anon_asrbn.yaml) A fast system based on vector quantized acoustic bottleneck, pitch, and one-hot speaker representations and  a HiFi-GAN speech synthesis model.
+
+      - #### ⚠ [Anonymization using x-vectors and a neural source-filter model](https://www.isca-archive.org/interspeech_2020/srivastava20_interspeech.pdf): **B1**
```

Comment:
This is not the correct paper to cite.
Please modify the link to: https://www.isca-archive.org/ssw_2019/fang19_ssw.html
```
F. Fang, X. Wang, J. Yamagishi, I. Echizen, M. Todisco, N. Evans,
and J.-F. Bonastre, “Speaker anonymization using X-vector and
neural waveform models,” in Speech Synthesis Workshop, 2019,
pp. 155–160.
```

Reply:
No, the replacement you propose is infact less relevant because this paper is different in many aspects from B1 (data, modules, x-vector anonymization approaches, evaluation, etc...).

Reply:
The data and evaluation from to which a system/baseline is apply on should not matter for the reference.
srivastava20_interspeech.pdf builds up an experiment on top of fang19_ssw to test many different target x-vector selection approach. 
It would be valid to cite srivastava20_interspeech.pdf if the target x-vector selection approach were from srivastava20_interspeech.pdf but it is not the case, the target speaker x-vector selection approach is from fang19_ssw.
To resume. Given the aspects you refer ((data, modules, x-vector anonymization approaches, evaluation, etc...))
For the data, the citation should be the VPC2020 eval plan.
For the modules, it should be a combination of the fist eval plan and fang19_ssw. (fang19_ssw for reseach, eval plan technical work for the released code)
For the x-vector anonymization approaches, it is fang19_ssw.
For the data, the citation should be the VPC2020 eval plan.

Please indicate were the srivastava20_interspeech paper has anything to do with this pipeline?

Furthermore, in the https://www.voiceprivacychallenge.org/docs/VoicePrivacy_2024_Eval_Plan_v2.0.pdf we say:
```
B1 is based on the voice anonymization method proposed in [47]

[47] F. Fang, X. Wang, J. Yamagishi, I. Echizen, M. Todisco, N. Evans, and J.-F. Bonastre, “Speaker
anonymization using x-vector and neural waveform models,” in Speech Synthesis Workshop, 2019, pp.
155–160.
```